apps/payment/transaction_constants.py

```diff
@@ -0,0 +1,12 @@
+class TransactionSource:
```

Comment:
https://docs.djangoproject.com/en/3.0/ref/models/fields/#field-choices-enum-types

Reply:
I think we could just have that as part of a greater refactor